precode.sh

```diff
@@ -6,38 +6,56 @@
 #   dir2
 #   dir3
 #       dir4
+mkdir -p task/dir1 task/dir2 task/dir3 task/dir3/dir4
```

Comment:
Флаг `-p` как раз позволяет не создавать промежуточные папки отдельно
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```